internal/bminventory/inventory.go

```diff
@@ -1973,10 +1959,28 @@ func (b *bareMetalInventory) updateClusterData(ctx context.Context, cluster *com
 		}
 	}
 
+	if params.ClusterUpdateParams.MachineNetworkCidr != nil && common.IsSingleNodeCluster(cluster) {
+		machineCidr = swag.StringValue(params.ClusterUpdateParams.MachineNetworkCidr)
+		if err = network.VerifySubnetCIDR(machineCidr); err != nil {
```

Comment:
Didn't we agree to remove this check?

Reply:
this check only validates that machine cidr is ok as machine cidr itself. for example 1.2.3.4/24 is not valid one